ci-tools/fpga-boss/README.md

```diff
@@ -68,7 +89,12 @@ $ sudo bash build.sh
 
 ```
 $ cd ci-tools/fpga-boss
-$ cargo run -- --zcu104 1-14.3 --sdwire 1-14.6 flash ../fpga-image/out/image.img
+# SDwire
+$ cargo run -- --zcu104 1-14.3 --sdmux_model sdwire --sdmux_id 1-14.6 flash ../fpga-image/out/image.img
```

Comment:
I think it's is enough to add a separate flag for `usbsdmux` to avoid a breaking change.

E.g. keep `--sdwire` as is, and add a `--usbsdmux` that takes a  `UsbSDMux` ID.

Reply:
I see your point, which is fair, about not breaking backward compatibility. The reason I added the `sdmux_model` flag was to make it ready for possible further models to be added in the future. Then one would have to add a new flag for each new SDMux, which, in my opinion, can be quite confusing.

Reply:
I agree this approach scales better but I think we can punt on it for now, until a third mux tool is introduced

Reply:
I talked to @MDr164 and he made a proposal I think could work out for us:
We could keep the `--sdwire` flag and also add the generic `--sdmux_model` and `--sdmux_id` flags, such that `--sdiwire <foo>` will internally be translated to `--sdmux_model sdwire --sdmux_id <foo>`. This way we keep the --sdwire flag for compatibility and can still introduce the new changes. What is your opinion on this?

Reply:
I think it's not too heavy of a lift to update the scripts, so if the preference is to add the generic flags let's move forward with it.

Another alternative:

Identify the correct backend by sequentially attempting to open the path in the `--sdwire` with each mux backend. That avoids a breaking change and keeps the interface generic.

Reply:
I think it should be okay but I don't want to bikeshed on this. I am okay with moving forward with the proposal.

> My personal favorite is the 'overshadowing' option.

Which option is this?